ui/src/components/yaml/index.tsx

```diff
@@ -254,6 +255,8 @@ const Yaml = (props: IProps) => {
       setStreaming(false)
```

Comment:
Potential null reference error: `getBoundingClientRect()` may return null if the element is not in the DOM or not visible. Consider adding a null check before accessing `top`.